### PR TITLE
Refactored gym.maps.js let's see if scutinizer like it

### PR DESCRIPTION
**Pull request by PoUpA**
## Description
Take this scruti !

## Motivation and Context
make scrutinizer green 🐶 

## How Has This Been Tested?
Localhost 
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by Obihoernchen** [comment]
🐕

File: core/json/.htaccess [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<Files "variables.json">
+    Order Deny,Allow
+    Deny from all
+    Allow from 127.0.0.1
+</Files>
```
Comment (Obihoernchen) [inline core/json/.htaccess:3]: Please use Apache 2.4 access control: http://httpd.apache.org/docs/current/howto/access.html
Or make an if module ... and include both.

**Review by Obihoernchen — CHANGES_REQUESTED** [review]

**Review by Obihoernchen — APPROVED** [review]

**Comment by Obihoernchen** [comment]
Is there someone with 2captcha who could test this please?

**Comment by PoUpA** [comment]
the result is this for me : 
image

**Comment by Obihoernchen** [comment]
I trust you and scruti :dog2:

**Pull request merged**